OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -405,6 +405,20 @@ public void OnInfiltrated(Actor a, LuaFunction func)
 			GetScriptTriggers(a).RegisterCallback(Trigger.OnInfiltrated, func, Context);
 		}
 
+		[Desc("Call a function when this actor is discovered by an enemy (doesn't work for neutral actors). " +
+			"The callback function will be called as func(Actor self, Player discoverer)")]
```

Comment:
`self` is ambiguous in this description (as well as the one below), and we make a point of not exposing this terminology to the API docs.